### DISCUSSION TITLE
How can I change the default color of time objects?

### DISCUSSION THREAD

**Discussion by kienerj** [Textualize/rich#3129] [Q&A]
As title says. By default the hr:min:sec part is in bright green and barley readable on a white background (juypter notebook if you are wondering about the white background)

**Comment by willmcgugan** [comment]
Search the docs for themes.

**Reply by kienerj** [reply]
Already did. Doesn't help

I can "wrap" the according part in an "inline style": 

    console.print(f"[green]{str(timedelta(seconds=elapsed))}[/green]")

Put not as a default for all occurences

**Comment by kienerj** [comment]
The issue here is that:

    print(str(timedelta(seconds=elapsed)))
    0:00:10

Gets recognized as ipv6 address. Hence setting the repr.ipv6 will fix the color:

    console = Console(theme=Theme({"repr.ipv6": "bold green"}))

Ideally there would be a different repr for timedelta.
